nservicebus/upgrades/5to6.md

```diff
@@ -610,4 +610,18 @@ The BSON serializer build into the core has been removed. Use the [Newtonsoft se
 
 [MSMQ Subscription Authorization](/nservicebus/msmq/subscription-authorisation.md) is now done by the `SubscriptionAuthorizer` delegate at configuration time and not the `IAuthorizeSubscriptions` interface.
 
-snippet: 5to6-MsmqSubscriptionAuthorizer
\ No newline at end of file
+snippet: 5to6-MsmqSubscriptionAuthorizer
+
+## Synchronous request-response
+
+Synchronous request-response capability, also known as the callbacks, has been removed from the `NServiceBus` package in V6. There is a separate `NServiceBus.Callbacks` NuGet package that needs to be installed in the requester endpoint in order to use the feature. The responding endpoint does not need any additional packages.
```

Comment:
> has been removed

has been moved to a separate NuGet package called `NServiceBus.Callbacks`

Sounds better than getting rid of something.